src/editor-page.jsx

```diff
@@ -17,7 +17,9 @@ var ViewportResizer = require("./components/viewport-resizer.jsx");
 
 var EditorPage = React.createClass({
     propTypes: {
+        /* eslint-disable react/forbid-prop-types */
         answerArea: React.PropTypes.object,
```

Comment:
Similar to what I said in the previous commit, I think I'd rather these be `React.PropTypes.any` to show that we don't really know what's in them.